cmd/package.go

```diff
@@ -657,14 +657,14 @@ func (b *BootOptions) GetCmd() (string, error) {
 	if b.Cmd != "" { // Direct commandLine has highest priority (--run <commandLine>).
 		fmt.Println("Command line will be set based on --run parameter")
 		command = b.Cmd
-	} else if b.Boot != "" { // Configuration name has second-highest priority (--boot <customBoot>).
-		fmt.Println("Command line will be set based on --boot parameter")
+	} else if len(b.Boot) > 0 { // Configuration name has second-highest priority (--boot <customBoot>).
+		fmt.Println("Command line will be set based on --boot parameters")
 		command = runtime.BootCmdForScript(b.Boot)
 	} else if b.PackageDir != "" { // Default configuration in yaml has third-highest priority (config_set_default: <>).
 		if data, err := ioutil.ReadFile(filepath.Join(b.PackageDir, "meta", "run.yaml")); err == nil {
 			if cmdConf, err := runtime.ParsePackageRunManifestData(data); err == nil && cmdConf.ConfigSetDefault != "" {
 				fmt.Println("Command line will be set based on config_set_default attribute of meta/run.yaml")
-				command = runtime.BootCmdForScript(cmdConf.ConfigSetDefault)
+				command = runtime.BootCmdForScript(strings.Split(cmdConf.ConfigSetDefault, "->"))
```

Comment:
Why not simple comma-separated list of configs?

Reply:
I've changed to comma-separated. Originally I went with `->` because it was more obvious that they will be executed sequentially, but yes, comma is more standardized.